mcm_2017c/__main__.py

```diff
@@ -0,0 +1,19 @@
+from execute_data import get_data

```

Comment:
```suggestion
from .execute_data import get_data
```

必须使用相对导入，否则从不同路径运行这个模块的时候会出现问题。